playroom/components.tsx

```diff
@@ -336,6 +338,14 @@ export const PreviewTools: React.FC<PreviewToolsProps> = ({
                 )}
             </>
         );
+    }
+    if (hide) {
+        return (
+            <>
+                {children}
+                <div style={{display: 'none'}} />
```

Comment:
I don't get this. Why the display none? isnt `return children` enough?

Reply:
true, I never used return xD